src/main/java/com/targomo/client/api/request/config/RequestConfigurator.java

```diff
@@ -283,6 +283,7 @@ private static AggregationConfiguration buildAggregationConfigFromTravelOptions(
                 .type(travelOptions.getMultiGraphAggregationType())
                 .aggregationInputParameters(travelOptions.getMultiGraphAggregationInputParameters())
                 .filterValuesForSourceOrigins(travelOptions.getMultiGraphAggregationFilterValuesForSourceOrigins())
+                .mathExpression(travelOptions.getMultiGraphAggregationMathExpression())
```

Comment:
you also need to add getMultiGraphAggregationMathExpression somewhere in the if check of line 59-67 

could you also please still add preAggregationPipeline there too